kotlinx.fuzz.engine/src/main/kotlin/kotlinx/fuzz/reproduction/ReproducerTemplate.kt

```diff
@@ -0,0 +1,17 @@
+package kotlinx.fuzz.reproduction
+
+import com.squareup.kotlinpoet.CodeBlock
+import com.squareup.kotlinpoet.FunSpec
+import com.squareup.kotlinpoet.TypeSpec
+
+interface ReproducerTemplate {
```

Comment:
Again from the name, it feels unclear to me what `ReproducerTemplate` does and why it does not extend `CrashReproducer`. Sorry for getting nerdy with the names, but the logic flow really is not clear to me because of them

Reply:
Tried to fix it. If still unclear, contact me)